core/sql/exp/exp_clause_derived.h

```diff
@@ -1542,7 +1542,9 @@ enum conv_case_index {
   CONV_UTF8_F_UCS2_V                   =248,
 
   CONV_BLOB_BLOB                       =249,
-  CONV_BLOB_ASCII_F                    =250
+  CONV_BLOB_ASCII_F                    =250,
+
+  CONV_GBK_F_UTF8_V                    =251
```

Comment:
Let me add this support. I didn't realize what the '_F' means in the beginning. Hive does not have CHAR or VARCHAR, it only support 'string'. So I assume Trafodion treats it as VARCHAR always, but let me try to add support of CONV_GBK_F_UTF8_F as well.